README.md

```diff
@@ -313,7 +313,15 @@ configuration is supported at the moment.
 
 Similar to `controller`, the `parent` references the connection profile in the ansible
 role.
+Here is a way to specify the VLAN ID
```

Comment:
Similar to `controller`, the `parent` references the connection profile in the ansible
role. The vlan ID can be specified by using nested `vlan` setting, the valid vlan ID 
value ranges from 0 to 4094. Here is how to specify the VLAN ID: 

```yaml
type: vlan
vlan:
 id: 6
```

Can you change to above?

Reply:
This is not resolved yet.